fluent-bundle/examples/external_arguments.rs

```diff
@@ -20,23 +20,26 @@ unread-emails =
         .expect("Failed to add FTL resources to the bundle.");
 
     let mut args = HashMap::new();
-    args.insert("name", FluentValue::from("John"));
+    args.insert("name".to_string(), FluentValue::from("John"));
 
-    match bundle.format("hello-world", Some(&args)) {
-        Some((value, _)) => println!("{}", value),
-        _ => println!("None"),
-    }
+    let msg = bundle.get_message("hello-world").expect("Message exists");
+    let mut errors = vec![];
+    let pattern = msg.value.expect("Message has a value");
```

Comment:
`expect()` takes the description of the _error scenario_, rather than of the success one :)